# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaDecoder.cpp
# Commit: 3e647187e78b
# Full Hash: 3e647187e78b9f74a372031572e4b0a502f72658
# Author: alwu <alwu@mozilla.com>
# Date: 2022-11-05 21:53:50
# Regressor Bug: 1262276
# File Overlap Count: 1
# Description:
#   Bug 1262276 - part2 : support seamless video looping. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D159126
# ==============================================================================

diff -r 1df4774b2c53 -r 3e647187e78b dom/media/MediaDecoder.cpp
--- a/dom/media/MediaDecoder.cpp	Sat Nov 05 02:23:28 2022 +0000
+++ b/dom/media/MediaDecoder.cpp	Sat Nov 05 02:23:29 2022 +0000
@@ -963,8 +963,7 @@
     double aPrevPos, double aCurPos) const {
   MOZ_ASSERT(NS_IsMainThread());
   // If current position is earlier than previous position and we didn't do
-  // seek, that means we looped back to the start position, which currently
-  // happens on audio only.
+  // seek, that means we looped back to the start position.
   const bool notSeeking = !mSeekRequest.Exists();
   if (mLooping && notSeeking && aCurPos < aPrevPos) {
     return PositionUpdate::eSeamlessLoopingSeeking;